persist/log.go

```diff
@@ -0,0 +1,30 @@
+package persist
+
+import (
+	"log"
+	"os"
+)
+
+// FileLogger is a wrapper for the standard library logger that enforces logging
+// into a file with the Sia-standard settings.
```

Comment:
The two main advantages of the logger are: 1) it opens files in append mode, and 2) it provides a `Close` method. These aren't really Sia-specific features.